### ISSUE TITLE
Is this project stil maintained?

### ISSUE THREAD

**Issue by GustavoOS** [Edgio/js-ectoken#1]
On NPM, the last publish was made 6 years ago, despite the recent changes to the main branch of this repository.

**Comment by tmountjr** [comment]
Hi @GustavoOS - this is a different package from the one that's on NPM. We're working on transitioning that repo (https://github.com/hattan/ectoken-nodejs) to the code here...I'm hoping it won't be longer than a few more days. This project is much newer and should be actively maintained going forward.

**Comment by tmountjr** [comment]
Functionally, https://github.com/Edgio/js-ectoken/releases/tag/v2.0.0-beta.1 should work the same as the package currently on NPM (as of May 28) with one exception - the `encrypt` and `decrypt` methods are now asynchronous.

**Comment by GustavoOS** [comment]
> Functionally, https://github.com/Edgio/js-ectoken/releases/tag/v2.0.0-beta.1 should work the same as the package currently on NPM (as of May 28) with one exception - the `encrypt` and `decrypt` methods are now asynchronous.

is it published on npm?

**Comment by GustavoOS** [comment]
![image](https://github.com/Edgio/js-ectoken/assets/15959001/6510fe76-2b2c-4639-b255-9fe658863a74)

**Comment by tmountjr** [comment]
Not yet - I've passed this over to the team who owns the NPM account, and they'll have to set that up from here. If you want to check it out before it is published, though, you can install it this way:

```
$ npm install git+https://git@github.com/Edgio/js-ectoken.git#v2.0.0-beta.1
```

I'll make a note to myself to respond back here once it has been published (probably as v2.0.0, no beta tag).

**Comment by GustavoOS** [comment]
Thanks :)

**Comment by GustavoOS** [comment]
@tmountjr I see there is a new package in npm registry @edgio/ectoken. Wouldn't it be the case to deprecate the ectoken?

**Comment by tmountjr** [comment]
@GustavoOS this repo is now published as v2.0.1 to npm (https://npmjs.com/package/@edgio/ectoken/v/2.0.1). I'll talk to the team about the previous package.

**Comment by tmountjr** [comment]
Since this is now a published package I'll go ahead and close out this issue.
